cscs-checks/microbenchmarks/stream/stream.py

```diff
@@ -125,7 +133,7 @@ def prepare_test(self):
         try:
             self.reference = self.stream_bw_reference[envname]
         except KeyError:
-            pass
+            self.reference = self.stream_bw_reference['*']
```

Comment:
Simply select the references from one of the known environments. The framework will pick then the units automatically. The problem was in this issue that in the case of an unknown programming environment, `reference` was empty and and the framework could not infer the units. Just picking any of the known programming environments here, it will do the trick.